apps/site/assets/ts/__v3api.d.ts

```diff
@@ -242,6 +242,9 @@ export interface Service {
   typicality: ServiceTypicality;
   valid_days: DayInteger[];
   name: string;
+  rating_start_date: string;
+  rating_end_date: string;
```

Comment:
Based on the logic and typespecs in `Service`, shouldn't these two be `string | null`?

Reply:
Should it? I'm trying to reuse the same logic for start_date and end_date.

Reply:
Yep, these can be null, often on ferries or commuter rail routes 🙃